COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -1000,19 +868,19 @@ zowe plugins install <name>@<distTag>
 For example,
 
 ```
-zowe plugins install @zowe/cics-for-zowe-cli@zowe-v1-lts
+zowe plugins install @zowe/cics-for-zowe-cli@zowe-v3-lts
 ```
 
 Multiple plug-ins can be installed in a single command. For example, to install all Zowe CLI plug-ins available from the Zowe organization, you could issue:
 
 ```{.bash}
-zowe plugins install @zowe/cics-for-zowe-cli@zowe-v1-lts @zowe/ims-for-zowe-cli@zowe-v1-lts @zowe/mq-for-zowe-cli@zowe-v1-lts @zowe/zos-ftp-for-zowe-cli@zowe-v1-lts @zowe/db2-for-zowe-cli@zowe-v1-lts
+zowe plugins install @zowe/cics-for-zowe-cli@zowe-v3-lts @zowe/ims-for-zowe-cli@zowe-v3-lts @zowe/mq-for-zowe-cli@zowe-v3-lts @zowe/zos-ftp-for-zowe-cli@zowe-v3-lts @zowe/db2-for-zowe-cli@zowe-v3-lts
```

Comment:
The IMS plugin seems to be [deprecated](https://www.npmjs.com/package/@zowe/ims-for-zowe-cli?activeTab=versions).

Reply:
Yes, we should remove the IMS plug-in from the list. It wasn't used in any of the exercises so we are okay from that perspective.